akka-actor-typed-tests/src/test/scala/akka/actor/typed/ExtensionsSpec.scala

```diff
@@ -151,17 +151,30 @@ class ExtensionsSpec extends TypedAkkaSpec {
         instance1 should be theSameInstanceAs instance2
       }
 
-    "not create an extension multiple times when using the ActorSystemAdapter" in {
+    "load registered typed extensions eagerly even for untyped system" in {
       import akka.actor.typed.scaladsl.adapter._
       val untypedSystem = akka.actor.ActorSystem()
       try {
-
         val before = InstanceCountingExtension.createCount.get()
         InstanceCountingExtension(untypedSystem.toTyped)
-        val ext = InstanceCountingExtension(untypedSystem.toTyped)
         val after = InstanceCountingExtension.createCount.get()
 
-        (after - before) should ===(1)
+        // should have been loaded even before it was accessed in the test because InstanceCountingExtension is listed
+        // as a typed library-extension in the config
```

Comment:
check the count before system was created as well, and see that system creation triggered one instantiation of it?

Reply:
Thanks good suggestion, changed.